app.js

```diff
@@ -0,0 +1,8 @@
+function onoff() {
+  let element = document.getElementById("getid");
+    element.classList.toggle("active");
+    element.removeEventListener('active')

```

Comment:
I guess that we have some mistake here. Please take a look at documentation of the removeEventListener method.